tests/src/app/testqgsmergeattributesdialog.cpp

```diff
@@ -150,6 +150,78 @@ class TestQgsMergeattributesDialog : public QgsTest
       QVERIFY( !dialog.mergedAttributes().at( 0 ).isValid() );
       QCOMPARE( dialog.mergedAttributes().at( 1 ), 22 );
     }
+
+    void testWithHiddenField()
+    {
+      // Create test layer
+      QgsVectorFileWriter::SaveVectorOptions options;
+      QgsVectorLayer ml( "LineString", "test", "memory" );
+      QVERIFY( ml.isValid() );
+
+      QgsField notHiddenField( QStringLiteral( "not_hidden" ), QVariant::Int );
+      QgsField hiddenField( QStringLiteral( "hidden" ), QVariant::Int );
+      // hide the field
+      ml.setEditorWidgetSetup( 1, QgsEditorWidgetSetup( QStringLiteral( "Hidden" ), QVariantMap() ) );
+      QVERIFY( ml.dataProvider()->addAttributes(
```

Comment:
Nit: this seems pretty unreadable to me, I think it should fit on a single line.

Reply:
Thanks for pointing out, it is indeed better formatted as single line.